src/lib/y2partitioner/actions/controllers/btrfs_subvolume.rb

```diff
@@ -126,15 +126,11 @@ def exist_subvolume?
           !new?(subvolume)
         end
 
-        # Whether the filesystem already has a subvolume on disk with the given path
+        # Whether the filesystem already has a subvolume with the given path
         #
         # @return [Boolean]
         def exist_path?(path)
-          subvolume = filesystem.btrfs_subvolumes.find { |s| s.path == path }
-
-          return false unless subvolume
-
-          !new?(subvolume)
+          filesystem.btrfs_subvolumes.any? { |s| s.path == path }
```

Comment:
Do you know why in the past we only checked for subvolumes that already existed in disk?

Reply:
This is used for the validation of the widget that allows to enter the subvolume path. For some reason, we decided that creating a subvolume with an existing path should be possible (if the subvolume does not exist on disk). Maybe the old partitioner allowed that? Anyway, creating a subvolume was only a matter of selecting its path and nocow property. But now, the Btrfs subvolume form has more options. So, silently replacing the subvolume with the same path is not a good idea.